_case_studies/09_databricks_solution_accelerator.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Optimizing Promotional Offers using Causal Machine Learning
```

Comment:
Let's add databricks to the title.  Compared to other case studies on pywhy, the fact that this is implemented in databricks is a differentiator that should be front and center

Reply:
Good point. Which of the following do you prefer:

"Optimizing Promotional Offers using Causal Machine Learning in Databricks"
"Using PyWhy in Databricks"
"Databricks Causal Machine Learning Solution Accelerator"
"PyWhy in the Databricks ecosystem"

Or happy to take suggestions on title.

Reply:
I like the second one, "Using PyWhy in Databricks", at least until there are multiple examples :)